.github/workflows/shared-java-publish-versioned-package.yaml

```diff
@@ -18,6 +18,9 @@ on:
         type: string
         default: 'true'
 
+env:
```

Comment:
This is not wrong, but you don't have to set it as an environment variable to use it. You could just pass ${{ github.ref }} directly to the version number

Reply:
I'm thinking of keeping it there for readability since we already do the same for IMAGE_NAME for this file